.github/workflows/build_and_test.yml

```diff
@@ -47,13 +47,31 @@ on:
         required: false
         type: boolean
         default: true
+      IS_OPTIONAL:
+        required: false
+        type: boolean
+        default: false
+      REQUIRED_LABEL:
+        required: false
+        type: string
```

Comment:
Do you need `IS_OPTIONAL` or could you rely on `REQUIRED_LABEL` being defined to gather if the job has to be run conditionally?

Reply:
Nope, I was just having problems with the syntax. Its fixed now.